modin/experimental/cloud/rayscale.py

```diff
@@ -158,17 +160,108 @@ def _conda_requirements(self):
-        return setup_commands.replace(
+        setup_commands = setup_commands.replace(
             "{{CONDA_PACKAGES}}", " ".join(self._conda_requirements())
         )
 
+        return setup_commands.replace(
+            "{{INSTALL_MODIN_COMMAND}}", self._get_modin_install_command()
+        )
+
     @staticmethod
     def _get_python_version():
         major = sys.version_info.major
         minor = sys.version_info.minor
         micro = sys.version_info.micro
         return [f"python>={major}.{minor}", f"python<={major}.{minor}.{micro}"]
 
+    @staticmethod
+    def _git_state():
+        """
+        Suppose git in PATH.
+        """
+        import modin
+
+        cwd = os.path.dirname(modin.__file__)
+        git_branch_vv = subprocess.check_output(
+            ["git", "branch", "-vv"], cwd=cwd, encoding="utf-8"
+        ).strip("\n")
+        git_remote_v = subprocess.check_output(
+            ["git", "remote", "-v"], cwd=cwd, encoding="utf-8"
+        ).strip("\n")
+
+        local_branch = None
+        remote_branch = None
+        for line in re.split(r"\n+", git_branch_vv):
+            # * LOCAL_BRANCH HASH [REPO_ALIAS/REMOTE_BRANCH[: STATUS]] [Commit message]
+            if line.startswith("*"):
+                local_branch = re.split(r"\s+", line)[1]
+                if local_branch == "HEAD":
+                    # example case: (HEAD detached at modin/sync-modin-between-contexts)
+                    raise ValueError("You are in 'detached HEAD' state")
+                if "[" not in line:
+                    raise ValueError(
+                        f"local branch: [{local_branch}] does not track remote"
+                    )
+                dirty_remote_branch = re.split(r"\s+", line)[3]
+                # remove "(" and ":" or ")"
+                remote_branch = dirty_remote_branch[1:][:-1]
+                break
+
+        try:
+            repo_alias, remote_branch = remote_branch.split("/", maxsplit=1)
+        except ValueError:
+            raise ValueError(
+                f"The remote branch: [{remote_branch}] obtained from \
+                git: [{git_branch_vv}] for local_branch: [{local_branch}] is not valid"
+            )
+
+        repo_name = None
+        for line in re.split(r"\n+", git_remote_v):
+            # repo_alias repo_name (fetch/push)
+            if line.startswith(repo_alias):
+                repo_name = re.split(r"\s+", line)[1]
+                break
+
+        if repo_name is None:
+            raise ValueError(
+                f"git remote -v: [{git_remote_v}] doesn't include info for repo alias: [{repo_alias}]"
+            )
+
+        return repo_name, remote_branch
+
+    @staticmethod
+    def _get_modin_install_command():
+        from modin import __version__
+
+        if len(__version__.split("+")) == 1:
+            # example version: 0.8.0
+            return f"conda install --yes --override-channels -c intel/label/validation \
```

Comment:
```suggestion
            return f"conda install --yes -c intel/label/validation \
```
same about `--override-channels`